core/worker.go

```diff
@@ -846,10 +846,11 @@ func (w *worker) prepareWork(genParams *generateParams, block *types.Block) (*en
 // fillTransactions retrieves the pending transactions from the txpool and fills them
 // into the given sealing block. The transaction selection and ordering strategy can
 // be customized with the plugin in the future.
-func (w *worker) fillTransactions(interrupt *int32, env *environment) {
+func (w *worker) fillTransactions(interrupt *int32, env *environment, block *types.Block) {
 	// Split the pending transactions into locals and remotes
 	// Fill the block with all available pending transactions.
-	pending, err := w.txPool.TxPoolPending(true)
+	etxSet := rawdb.ReadEtxSet(w.hc.bc.db, block.Hash(), block.NumberU64())
```

Comment:
Need to check if the `etxSet` was `nil` here. `TxPoolPending()` will just ignore it if `nil`, but `nil` indicates an error that should be handled

Reply:
Done, https://github.com/dominant-strategies/go-quai/pull/481/commits/97c6e1620aaa4151554ab27b79e4afa610e30ae4